test/performance/benchmarks_test.go

```diff
@@ -148,7 +146,17 @@ func TestBenchmarkBurstZeroToN(t *testing.T) {
 func TestBenchmarkBurstNto2N(t *testing.T) {
 	for _, load := range loads {
 		t.Run(fmt.Sprintf("N%d", load), func(t *testing.T) {
-			runTest(t, app, float64(load), []float64{1, 2})
+			// Steady ramp up from 0 to N, then burst to 2N
+			zeroToNSteadyPacer := vegeta.SinePacer{
```

Comment:
You need a new pacer here, I think.

Reply:
Which pacer do you suggest to use? I've considered `ConstantPacer` with N QPS as the first step, but then I think this steadily increasing one might be better.

Reply:
Same as above.